lib/ClangImporter/ImportMacro.cpp

```diff
@@ -334,6 +345,29 @@ static ValueDecl *importMacro(ClangImporter::Implementation &impl,
         tokenI[3].is(clang::tok::r_paren)) {
       return importStringLiteral(impl, DC, macro, name, tokenI[2],
                                  MappedStringLiteralKind::CFString, ClangN);
+    // Check for literals with a cast, e.g. "(cpu_type_t) 1"
+    } else if (tokenI[0].is(clang::tok::l_paren) &&
+               tokenI[1].is(clang::tok::identifier) &&
```

Comment:
I assume one would propagate down the "known cast type" and not parse a second cast? I do think it would be a lot cleaner that way.